### PR TITLE
Allow to export the timings as JSON.

### PR DESCRIPTION
**Pull request by Carreau**
Also include the status of tests in dumps cause a faster commit that
fails, or a slower commit that used to be skipped can always happen.

--- 

Typically I want to build a stream graph of time taken by tests across a repository.

### CODE REVIEW

**Comment by skudriashev** [comment]
@Carreau, please, fix build. Thanks.

File: nosetimer/plugin.py [hunk 1]
```diff
@@ -236,6 +248,10 @@ def options(self, parser, env=os.environ):
         parser.add_option("--timer-top-n", action="store", default="-1",
                           dest="timer_top_n", help=_help)
 
+        parser.add_option("--timer-json-file", action="store",
+                default=None,
+                          dest="json_file", help=_help)
```
Comment (skudriashev) [inline nosetimer/plugin.py]: You use `_help` here from the above option. Please, write appropriate one.

**Comment by skudriashev** [comment]
Also, please remove `.` from the commit message.

File: nosetimer/plugin.py (hunk 1 above)
Comment (Carreau) [inline nosetimer/plugin.py]: Oops.

**Comment by Carreau** [comment]
All comments addressed, I think.

**Comment by skudriashev** [comment]
@mahmoudimus, please have a look if you are fine with this change. Thanks.

**Comment by mahmoudimus** [comment]
👍 merge it

**Comment by skudriashev** [comment]
@Carreau, thanks for the contribution!

**Comment by Carreau** [comment]
> @Carreau, thanks for the contribution!

No problem, I was looking to write something similar, you simplified me the task.

Thanks !

**Pull request merged**